samples/subsys/usb/console/app.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,console = &cdc_acm_uart0;
+	};
+
+	cdc_acm_uart0: uart {
+		status = "okay";
```

Comment:
You can leave out the 'status = "okay";' if you want, since that is the default value. But it's your decision.